engine/buildkit/filesync.go

```diff
@@ -72,7 +72,7 @@ func (c *Client) LocalImport(
 	// blobSource.CacheKey for more context
 	copyLLB := llb.Scratch().File(
 		llb.Copy(localLLB, "/", "/"),
-		llb.WithCustomNamef(localName),
+		llb.WithCustomNamef("copy %s", localName),
```

Comment:
shall  we add the `[internal]` prefix so it doesn't appear in cloud by default unless the `debug`view is set? https://github.com/marcosnils/dagger/blob/e5feaea7f1d6eb01d834e83b180fc3daed3463ea/engine/buildkit/progrock.go#L147. 

As this seems like effectively an `internal` thing we do, I don't think it's valuable to bubble it up to the users.

Reply:
Good call, I wasn't aware of our behavior here! Have done this.